awx/main/notifications/email_backend.py

```diff
@@ -19,6 +19,12 @@ class CustomEmailBackend(EmailBackend):
                        "sender": {"label": "Sender Email", "type": "string"},
                        "recipients": {"label": "Recipient List", "type": "list"},
                        "timeout": {"label": "Timeout", "type": "int", "default": 30}}
+
+    DEFULT_SUBJECT = "{{ job_friendly_name }} #{{ job.id }} '{{ job.name }}' {{ job.status }}: {{ url }}"
```

Comment:
Also picky, but `DEFULT_SUBJECT` has a typo.

Reply:
(throughout these backends, due to copy-pasta)

Reply:
👍 fixed